config/routes.rb

```diff
@@ -1,9 +1,17 @@
 Rails.application.routes.draw do
+  root "photos#index"
 
-  root photos#index 
+  devise_for :users
   
-  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
+  resources :comments
+  resources :follow_requests
+  resources :likes
+  resources :photos
 
-  # Defines the root path route ("/")
-  # root "articles#index"
+  get ":username/liked" => "users#liked", as: :liked
+  get ":username/feed"
+  get ":username/followers"
+  get ":username/following"
```

Comment:
Either comment out these lines, or finish pathing them
```ruby
  get ":username/feed" => "users#feed", as: :feed
   get ":username/followers" => "users#followers", as: :followers
   get ":username/followers" => "users#following", as: :following
```

If you do finish pathing them, you'll also need to create the appropriate methods in `users_controller.rb`